manifest.yaml

```diff
@@ -12,10 +12,12 @@ modules:
 postRunCommand:
   - name: asdf install
     command: ./scripts/devbase.sh; source .bootstrap/shell/lib/asdf.sh; asdf_install
+  - name: gogenerate
```

Comment:
Unless `gogenerate` runs `go mod tidy` this will sometimes fail, which is why we have this ordering :/

Reply:
Yeah, but I thought we went back and forth, and finally settled on doing it this way since `gogenerate` can sometimes add dependencies.

Actually, what if we just run `go mod tidy` twice. That should work, right?

Reply:
Let's just try this for now and see what happens :D